spec/unit/bsd/hostname_if/inet_spec.rb

```diff
@@ -16,14 +16,49 @@
   end
 
   describe 'process' do
-    it "should yield the the dynamic addressing is specified for all AF" do
-      a = [
-        'dhcp',
-        'rtsol',
-      ]
-      expect {|b|
-        PuppetX::BSD::Hostname_if::Inet.new(a).process(&b)
-      }.to yield_successive_args('dhcp','rtsol')
+    context 'On OpenBSD 5.6' do
+      it "should yield the the dynamic addressing is specified for all AF with rtsol" do
+        a = [
+          'dhcp',
+          'rtsol',
+        ]
+	expect(Facter).to receive(:value).with('kernelversion').at_least(:once).and_return('5.6')
```

Comment:
This looks to need another space in front, as do the rest of the similar expect lines.